### ISSUE TITLE
Broken description text in Pop!_Shop

### ISSUE THREAD

**Issue by Juaan77** [pop-os/shop#224]
**Distribution (run `cat /etc/os-release`):**
 Pop!_OS 20.04
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Pop!_Shop

**Issue/Bug Description:**
Description text for Pop!_Shop apps is broken/displays badly. It overlaps itself when you scroll down the app page.

**Steps to reproduce (if you know):**
1- Open Pop!_Shop
2- Choose any app
3- Scroll down

**Expected behavior:**
Text should display correctly.

**Other Notes:**
It started to look like this like a month ago after an update. Looks the same no matter it's a .deb or flatpak. I'm adding some screenshots below.

![Captura de pantalla de 2020-09-29 11-06-08](https://user-images.githubusercontent.com/61911911/94570618-6a0a7000-0245-11eb-9323-07cb30be5f0c.png)
![Captura de pantalla de 2020-09-29 11-06-22](https://user-images.githubusercontent.com/61911911/94570626-6bd43380-0245-11eb-9190-9738ddea3080.png)
![Captura de pantalla de 2020-09-29 11-09-11](https://user-images.githubusercontent.com/61911911/94570629-6c6cca00-0245-11eb-947d-965d9271da6f.png)

**Comment by TechnologyMan101** [comment]
Hmmm. It seems fine to me.

![Screenshot from 2020-09-29 10-54-59](https://user-images.githubusercontent.com/65311278/94597310-44c83280-0242-11eb-9e9f-5715f741208b.png)

**Comment by Juaan77** [comment]
Yeah I really don't know what's going on, it only happens on that specific type of text of the pop shop.

**Comment by CaptainJawZ** [comment]
Happens to me too, also since I updated to pop 20.10

**Comment by erikeah** [comment]
Happens to me too. 20.04 LTS
